### ISSUE TITLE
Is it possible to allow a user to minimize or resize an IDeskBand

### ISSUE THREAD

**Issue by amtopel** [stefankueng/tools#49]
Hi, I had a general question about Explorer toolbars and was wondering if you could help.  Do you know if it's possible to hide/minimize/resize a toolbar dynamically?

I've been experimenting and can't figure out if it's possible.  Simply hiding the toolbar's `hwnd `does not work.  In that case, the toolbar will be hidden, but the host window will still be there, so there will be just a blank rectangle.

So I tried `IOleCommandTarget::Exec` with `DBID_BANDINFOCHANGED`, and I was able to successfully trigger a new call to `GetBandInfo`.  But it appears that providing new values won't change the host position.  It seems that the ones provided on startup always govern, though maybe I'm doing something wrong.

Do you know the trick to get something like this to work?  Thanks for any help.

**Comment by stefankueng** [comment]
I don't think this is possible. Simply because MS often blocks apps and especially plugins to change what a user can configure. So when the user places your deskband somewhere it should maybe stay there.

**Comment by amtopel** [comment]
Okay, thanks.  But then what's the purpose or effect of calling `IOleCommandTarget::Exec` with `DBID_BANDINFOCHANGED`?
